<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Noman" failures="0" tests="4" name="Default test" time="0.026" errors="0" timestamp="2022-06-28T16:47:20 BDT">
  <testcase classname="common.BaseTest" name="@BeforeSuite launchBrowser" time="0.022">
    <failure type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
at io.github.bonigarcia.wdm.config.Config.defaultOsName(Config.java:346)
at io.github.bonigarcia.wdm.config.Config.<init>(Config.java:103)
at io.github.bonigarcia.wdm.WebDriverManager.<init>(WebDriverManager.java:189)
at io.github.bonigarcia.wdm.managers.FirefoxDriverManager.<init>(FirefoxDriverManager.java:41)
at io.github.bonigarcia.wdm.WebDriverManager.firefoxdriver(WebDriverManager.java:212)
at common.BaseTest.launchBrowser(BaseTest.java:16)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NumberFormatException: multiple points
at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:1914)
at java.base/jdk.internal.math.FloatingDecimal.parseFloat(FloatingDecimal.java:122)
at java.base/java.lang.Float.parseFloat(Float.java:476)
at org.apache.commons.lang.SystemUtils.getJavaVersionAsFloat(SystemUtils.java:1117)
at org.apache.commons.lang.SystemUtils.<clinit>(SystemUtils.java:817)
... 25 more
]]>
    </failure>
  </testcase> <!-- @BeforeSuite launchBrowser -->
  <testcase classname="ui.ReportsTest" name="reporterTest1" time="0.0">
    <skipped/>
  </testcase> <!-- reporterTest1 -->
  <testcase classname="ui.ReportsTest" name="reporterTest2" time="0.0">
    <skipped/>
  </testcase> <!-- reporterTest2 -->
  <testcase classname="ui.ReportsTest" name="reporterTest3" time="0.0">
    <skipped/>
  </testcase> <!-- reporterTest3 -->
  <testcase classname="ui.ReportsTest" name="reporterTest4" time="0.0">
    <skipped/>
  </testcase> <!-- reporterTest4 -->
</testsuite> <!-- Default test -->
